# generated by patch-package 6.4.10
#
# command:
#   npx patch-package @vscode/test-electron
#
# declared package:
#   @vscode/test-electron: ^2.1.5
#
diff --git a/node_modules/@vscode/test-electron/out/runTest.js b/node_modules/@vscode/test-electron/out/runTest.js
index edf5f22..2206469 100644
--- a/node_modules/@vscode/test-electron/out/runTest.js
+++ b/node_modules/@vscode/test-electron/out/runTest.js
@@ -62,7 +62,13 @@ async function innerRunTests(executable, args, testRunnerEnv) {
         const fullEnv = Object.assign({}, process.env, testRunnerEnv);
         const cmd = cp.spawn(executable, args, { env: fullEnv });
         cmd.stdout.on('data', d => process.stdout.write(d));
-        cmd.stderr.on('data', d => process.stderr.write(d));
+		let debug = ''
+        cmd.stderr.on('data', d => {
+			const str = String(d)
+			if (str.includes('Failed to connect to the bus')) return
+			if (str.match(/(rejected promise)/)) process.stderr.write(d);
+			else debug += `\n${str}`
+        });
         cmd.on('error', function (data) {
             console.log('Test error: ' + data.toString());
         });
@@ -77,6 +83,7 @@ async function innerRunTests(executable, args, testRunnerEnv) {
                 reject(signal);
             }
             else if (code !== 0) {
+				console.log('debug', debug)
                 reject('Failed');
             }
             else {
